File: 1482-how-many-numbers-are-smaller-than-the-current-number/1482-how-many-numbers-are-smaller-than-the-current-number.cpp
class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) {
        vector<int> newNums;
        for (int i=0; i<nums.size(); i++){
            int count = 0;
            for (int j=0; j<nums.size();j++){
                if (nums[i] > nums[j]){
                    count++;
                }
            }
            newNums.push_back(count);
        }
        return newNums;
    }
};

File: 1482-how-many-numbers-are-smaller-than-the-current-number/1482-how-many-numbers-are-smaller-than-the-current-number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        new_nums = []
        count = 0
        for i in range(len(nums)):
            count = 0
            for j in range(len(nums)):
                if nums[j] < nums[i]:
                    count+=1
            new_nums.append(count)
                
        return new_nums